next-app/BUILD.bazel

```diff
@@ -0,0 +1,61 @@
+load("@npm//:next/package_json.bzl", next_bin = "bin")
```

Comment:
isn't this load unused now?

Reply:
still in  use for the start ("serve") target which doesn't suffer from the same problem as the build target
```
next_bin.next_binary(
    name = "start",
    args = ["start"],
    data = [
        ":node_modules",
        ":srcs",
    ],
)
```